template/{{app_name}}/tests/src/logging/test_audit.py

```diff
@@ -146,6 +146,24 @@ def test_audit_hook(
         assert_record_match(record, expected_record)
 
 
+def test_get_addr_info(init_audit_hook, caplog: pytest.LogCaptureFixture):
```

Comment:
this test is redundant with `test_audit_hook`, there's already a use case for socket.getaddrinfo (see https://github.com/navapbc/template-application-flask/blob/12c5d00e09298597b7767c78ab0686f749af2536/template/%7B%7Bapp_name%7D%7D/tests/src/logging/test_audit.py#L96-L101)